### PR TITLE
update with new css/fonts

### PR DESCRIPTION
**Pull request by KMontag42**
newest possible version of the template/assets!

woo!

### CODE REVIEW

File: assets/alt-theme.css [hunk 1]
```diff
@@ -0,0 +1,187 @@
+body {
+    background: #1b1b1b;
+    padding: 0;
+    margin: 0;
+}
+
+.sessionm-app * {
+    font-family: arial !important;
```
Comment (pygeek) [inline assets/alt-theme.css]: Small but `important!` necessary here? Also, how about fallback? Consider:
`font-family: Arial, Helvetica, sans-serif;`

File: assets/alt-theme.css [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+}
+#oAuth {
+    width: 230px;
+    float: right;
```
Comment (pygeek) [inline assets/alt-theme.css]: Any reason why we're using floats instead of flex?

**Comment by KMontag42** [comment]
all of this is the result of the `build` process from the core repository itself.

it should not be the responsibility of this repo to moderate the output of that process. Instead this repo should simply contain the output of the most up to date master branch from the core repo.

**Comment by pygeek** [comment]
@KMontag42 I had spoken to @adeka about updating https://github.com/sessionm/sessionm-ewc . Builds should be placed on there versioned via tag. One of the reasons for this is to be able to use it as a git submodule for both the demo and this repository to avoid copying code between repositories—because you're right, it's not this repository's responsibility to maintain this code.

**Comment by KMontag42** [comment]
totally agreed. i will work on that after travis ^_^

**Pull request merged**